manubot/cite/handlers.py

```diff
@@ -25,6 +25,25 @@
     "manubot.cite.wikidata.Handler_Wikidata",
 ]
 
+_infer_prefix_patterns = {
+    ("doi", "accession_pattern"),
+    ("doi", "shortdoi_pattern"),
+    ("pmc", "accession_pattern"),
```

Comment:
`pmc:pmc4304851` and  `pmc4304851` both fail since PMC is lowercase. But with the former you get a warning:

```
## WARNING
Inspection of dealiased citekeys revealed potential problems:
pmc:pmc4304851 -- PubMed Central Identifiers must start with 'PMC'.
```